testing/matrix/linter/rules/roles/placement.go

```diff
@@ -316,7 +316,7 @@ func handleNestedRBACForUs(m utils.Module, object storage.StoreObject, shortPath
 
 	switch {
 	case strings.HasPrefix(objectName, localPrefix):
-		if namespace != m.Namespace {
+		if namespace != m.Namespace && namespace != "default" {
```

Comment:
Oh no 😞

Reply:
This condition must not be changed. `default` is considered a system namespace, so the rule's name must start with `d8:` prefix.

Reply:
btw, it is better to use v1.NamespaceDefault.

Reply:
fixed by proper way